src/main/java/com/textmessenger/model/entity/TemporaryToken.java

```diff
@@ -0,0 +1,40 @@
+package com.textmessenger.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.sql.Timestamp;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.sql.Timestamp'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=166&issues=AWWjtg57AJjEPbGf0_B4&open=AWWjtg57AJjEPbGf0_B4)

[ISSUE_KEY]: AWWjtg57AJjEPbGf0_B4